doc/source/api/solver/fieldinfo.rst

```diff
@@ -0,0 +1,98 @@
+.. _ref_field_info:
+
+Field info
+==========
+
+You can use field info objects to access Fluent field information.
+
+Accessing field info objects
+----------------------------
+
+In order to access field info, launch the fluent solver, and make field
+data available (for example, by reading case and data files):
+
+.. code-block:: python
+
+  >>> import ansys.fluent.core as pyfluent
+  >>> solver = pyfluent.launch_fluent(mode="solver")
+  >>> solver.file.read(file_type="case-dats", file_name=mixing_elbow_case_path)
+  >>> solver.solution.initialization.hybrid_initialize()
+
+
+The field info object is an attribute of the solver object:
+
+.. code-block:: python
+
+  >>> field_info = solver.field_info
+
+Sample requests
+---------------
+
+Here are the methods for requesting each type of field:
```

Comment:
@prmukherj "for requesting each type of field" is not quite accurate. E.g., `get_range` returns the range of values for a given field rather than an actual field.

Reply:
Thank you @seanpearsonuk, it was a mistake. Corrected it.